sci-geosciences/grass/grass-9999.ebuild

```diff
@@ -236,7 +236,7 @@ os.environ\[\"GRASS_PYTHON\"\] = \"${EPYTHON}\":" \
 
 	# set proper GISDBASE directory path in the demolocation .grassrc80 file
 	sed -e "s:GISDBASE\:.*$:GISDBASE\: ${gisbase}:" \
-		-i "${ED}"${gisbase}/demolocation/.grassrc80 || die
+		-i "${ED}"${gisbase}/demolocation/.grassrc81 || die
```

Comment:
Not something you _necessarily_ need to fix right now, but it'd be nice to use some manipulation of `${SLOT}` for this (e.g. `.${SLOT/./}`) instead, so we don't forget to update it.

Also, should probably update the comment above.

Reply:
Yes you are right, thanks.